code/node-server/package.json

```diff
@@ -32,6 +32,7 @@
     "express-rate-limit": "^2.11.0",
     "express-ws": "^3.0.0",
     "redis": "^2.8.0",
+    "supervisor": "^0.12.0",
```

Comment:
also we shouldn't rely on ppl having this node module installed globally on their machines. Fixed by adding `supervisor` as a dependency

![bitmoji](https://render.bitstrips.com/v2/cpanel/e44376a0-4919-4b1c-8207-e8c2953ac144-7c243825-53cb-4c20-a7d9-3fc8d332e86d-v1.png?transparent=1&palette=1&width=246)